### ISSUE TITLE
On Jupyter notebook: Model Training Progress Bar breaks when you stop the kernel

### ISSUE THREAD

**Issue by astarostap** [HazyResearch/metal#80]
Duplication steps (in a jupyter notebook):
1) In a cell: create an EndModel and start training it
2) click on the stop button while the model is training (the progress bars are correctly showing training is going on)
3) rerun the cell.
4) You'll see that the progress bar is now working and is printing multiple lines saying there's 0% progress.

@jdunnmon mentioned the solution is to use tqdm_notebook
https://stackoverflow.com/questions/42212810/tqdm-in-jupyter-notebook

**Comment by paroma** [comment]
This is taken care of in the hackathon branch: https://github.com/HazyResearch/metal/blob/7a8e7299a0b5813d323e21a61056074ea79fed52/metal/classifier.py#L18

**Comment by jdunnmon** [comment]
@paroma, yeah @SenWu showed me how to do this a bit ago, and this should be fixed.  Feel free to close if working for you.
